### PR TITLE
[AIRFLOW-6016] Remove redundant setting `remote_logging`

### PR DESCRIPTION
**Pull request by serkef**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-6016
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
NA
### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Trivial change

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

**Comment by OmerJog** [comment]
@serkef can you fix?
`airflow/configuration.py:462:1: E303 too many blank lines (3)`

**Comment by XD-DENG** [comment]
Personally I would give a -1 for this change, since I think it's necessary to keep this boolean configuration item as a "switch".

File: docs/howto/write-logs.rst [hunk 1]
```diff
@@ -157,9 +154,10 @@ First, to use the handler, ``airflow.cfg`` must be configured as follows:
     [core]
     # Airflow can store logs remotely in AWS S3, Google Cloud Storage or Elastic Search.
     # Users must supply an Airflow connection id that provides access to the storage
-    # location. If remote_logging is set to true, see UPDATING.md for additional
-    # configuration requirements.
-    remote_logging = True
+    # location. To set up correctly, see https://airflow.readthedocs.io/en/stable/howto/write-logs.html
```
Comment (ashb) [inline docs/howto/write-logs.rst:157]: General point: Don't link to readthedocs.io but to airflow.apache.org

**Comment by ashb** [comment]
Agreed. It's nice to easily turn it on/off without caring _which_ setting is in use.

**Pull request closed without merging**